src/content/guides/getting-started.md

```diff
@@ -202,7 +202,7 @@ You can also set it to 'none' to disable any default behavior. Learn more: https
 
 T> Your output may vary a bit, but if the build is successful then you are good to go. Also, don't worry about the warning, we'll tackle that later.
 
-Open `index.html` in your browser and, if everything went right, you should see the following text: 'Hello webpack'.
+Open `dist/index.html` in your browser and, if everything went right, you should see the following text: 'Hello webpack'.
```

Comment:
I think this will be more accessible by wider audience, what do you think?
```suggestion
Open `index.html` from the `dist` directory in your browser and, if everything went right, you should see the following text: 'Hello webpack'.
```

Reply:
Sure, you're totally right! That's better